Basic-Car-Maintenance/Shared/Odometer/Views/ReadingView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ReadingView.swift
+//  Basic-Car-Maintenance
+//
+//  https://github.com/mikaelacaron/Basic-Car-Maintenance
+//  See LICENSE for license information.
+//
+
+import SwiftUI
+
+struct ReadingView: View {
+    private let reading: OdometerReading
+    private let vehicleName: String?
+    
+    init(odometerReading: OdometerReading, vehicleName: String?) {
+        self.reading = odometerReading
+        self.vehicleName = vehicleName
+    }
```

Comment:
these variables don't need to be private, and by making them not private, we don't need to manually create an initializer

I fixed this